src/cli/utils.ts

```diff
@@ -284,3 +285,46 @@ export function getUploadFolderOptions(options: UploadFolderOptions): {
     maxConcurrentUploads: +(options.maxConcurrency ?? 1),
   };
 }
+
+/**
+ * Parse tags array from CLI input into Tag array
+ * Accepts format: ["name1", "value1", "name2", "value2"]
+ * @param tagsArr Array of alternating tag names and values
+ * @returns Array of {name: string, value: string} objects
+ */
+export function parseTags(
+  tagsArr?: string[],
+): { name: string; value: string }[] {
+  if (!tagsArr || tagsArr.length === 0) {
+    return [];
+  }
+
+  if (tagsArr.length % 2 !== 0) {
+    throw new Error(
+      'Invalid tags format. Tags must be provided in pairs of name and value.',
+    );
+  }
+
+  const tags: { name: string; value: string }[] = [];
+
+  for (let i = 0; i < tagsArr.length; i += 2) {
```

Comment:
No need to change this but another way to do this in fewer lines of code:
```
while(a.length) {
  const tag = { name: tagsArr.shift(), value: tagsArr.shift() };
  if (!(tag.name && tag.value)) {
    throw new Error(
        'Invalid tag format. Each tag must have both a name and value.',
      );
  }
  tags.push(tag);
}
```
While loops can be risky but the risks are pretty contained here.

Reply:
QUICK NOTE: Empty strings are falsey in js. Double check on the validity of empty tag names and values.